webui/react/src/utils/dom.ts

```diff
@@ -18,6 +18,26 @@ export const toRem = (px?: number | string): string => {
   return px;
 };
 
+/*
+ * Calculates Pixel from Rem based on base font size of 62.5%.
+ * This causes the ratio between REM to px to be 1 to 10.
+ * So 1rem becomes 10px.
+ */
+export const toPixel = (rem?: number | string): string => {
+  if (rem == null) return 'auto';
+  if (typeof rem === 'number') return `${rem * 10}px`;
+
+  const matches = rem.match(/(\d+\.?\d*)\s*(px|rem)/i);
```

Comment:
**suggestion:** make it also support something like `.5rem` where there isn't a digit in front of a decimal. Something like this (untested):
```
/(\d+\.?\d*|\.\d+)\s*(px|rem)/i
```

Reply:
great feedback, thanks!